src/molecule.py

```diff
@@ -246,48 +242,20 @@ def emit(self, record):
             'Fr','Ra','Ac','Th','Pa','U','Np','Pu','Am','Cm','Bk','Cf','Es','Fm','Md','No','Lr','Rf','Db','Sg','Bh','Hs','Mt']
 
 # Dictionary of atomic masses ; also serves as the list of elements (periodic table)
-#
```

Comment:
This needs to be reverted, perhaps by merging the master branch into this branch.